### ISSUE TITLE
Up, Down, Left and Right

### ISSUE THREAD

**Issue by ghost** [Bone-Project/BoneOS#118]
- [x] Left and Right: move around and edit the command
- [ ] Up and Down: you know that.

**Comment by amanuel2** [comment]
Up for next release

**Comment by double-fault** [comment]
For up and down, if you want to go above more than one command, you will have to wait for the memory subsytem and filesystem to come, as that is the only way we can store the command history and navigate through it.

**Comment by double-fault** [comment]
Just for updating in the progress: As I told earlier, this is going to take some while as we need to wait for the memory subsystem and filesystem to come. @amanuel2 is currently implementing the memory subsystem, and after that I will implement the filesystem. 

**TL;DR**, it's going to take some while before this is going to be implemented.

**Comment by double-fault** [comment]
Good news, I found a way to implement left arrow and right arrow. I  _guess_ I can implement it, but I still can't give you a 100% it will work. Though the Up and Down will still have to wait for the filesystem to come.

**Comment by double-fault** [comment]
I have finished the left arrow key, try it out. Doing right arrow key now..

**Comment by double-fault** [comment]
Finished the (left and right) arrow keys, only need to fix a couple of bugs in them. Once done, I'll checkmark the checkbox.

**Comment by double-fault** [comment]
All bugs are fixed, left and right arrow keys _should_ be functional. If you encounter any bugs, please post a new issue.
